R/google_sheets.R

```diff
@@ -27,6 +27,31 @@ updateGoogleSheet <- function(filepath, id, overwrite = FALSE){
   sheet <- googledrive::drive_update(file = googledrive::as_id(id), media = filepath)
 }
 
+#' API to upload data frame to Google Sheets.
+#'
+#' @export
+#' @param df - data frame
+#' @param type - either "newSpreadSheet", "overrideSpreadSheet", "newWorkSheet", and "appendToWorkSheet"
+#' @param spreadSheetName - name of the spread sheet (when creating a new spread sheet)
+#' @param spreadSheetId - sheet id (when updating an existing spread sheet)
+#' @param workSheet - name of the worksheet
+#'
```

Comment:
Add prerequisites for using this function. I think oauth needs to be set in advance.

Reply:
Added comment for pre-requisite.